Dockerfile.run

```diff
@@ -6,7 +6,8 @@ ARG pack_uid=1000
 ARG pack_gid=1000
 
 RUN groupadd heroku --gid ${pack_gid} && \
-  useradd heroku -u ${pack_uid} -g ${pack_gid} -s /bin/bash -d /workspace -m
+  useradd heroku -u ${pack_uid} -g ${pack_gid} -s /bin/bash -m
 
 LABEL io.buildpacks.stack.id="heroku-18"
 USER heroku
+ENV HOME /app
```

Comment:
I don't actually know if this will suffice. The problem will be the PHP buildpack.